definitions/divorce/json/State.json

```diff
@@ -339,5 +339,14 @@
     "Description": "Awaiting admin clarification",
     "DisplayOrder": 22,
     "TitleDisplay": "${[CASE_REFERENCE]}: ${D8PetitionerLastName} vs ${D8RespondentLastName}"
+  },
+  {
+    "LiveFrom": "02/07/2019",
+    "CaseTypeID": "DIVORCE",
+    "ID": "AwaitingAmendCase",
```

Comment:
You will also need to add authorisations for this state.
You will need "RU" (or "CRU") for caseworker-divorce-courtadmin_beta
then "R" for: citizen, caseworker-divorce-courtadmin, caseworker-divorce-solicitor, caseworker-divorce-systemupdate, caseworker-divorce-bulkscan, caseworker-divorce-courtadmin-la and caseworker-divorce-superuser

Reply:
I have messaged you on slack in regards to this comment. id love some further clarification on this.